test/integration/a_visitor_can_view_an_individual_item_page_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.